homeassistant/components/deako/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""The deako integration."""
+from __future__ import annotations
+
+import logging
+
+from pydeako.deako import Deako, FindDevicesTimeout
+from pydeako.discover import DeakoDiscoverer
+
+from homeassistant.components import zeroconf
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+_LOGGER: logging.Logger = logging.getLogger(__package__)
+
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up deako from a config entry."""
+    hass_data = hass.data.setdefault(DOMAIN, {})
+    if hass_data is None or not isinstance(hass_data, dict):
+        hass_data = {}
+        hass.data[DOMAIN] = hass_data
```

Comment:
Why do we need this?

Reply:
Hmmmm not needed anymore. I thought I needed it for something with discovery, but I don't anymore. Thanks for the comment!